trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/CatalogConnectorManager.java

```diff
@@ -133,8 +133,11 @@ private void loadMetalake() {
           LOG.error("Load Metalake {} failed.", usedMetalake, e);
         }
       }
-    } catch (Throwable t) {
-      LOG.error("Fatal errors when loading metalake", t);
+    } catch (OutOfMemoryError outOfMemoryError) {
```

Comment:
Thanks for helping out here. There's no need to catch OutOfMemoryError or InternalError explicitly, once that happens there's probably not much you can do and I'm not sure the extra logging adds anything. Throwable include Expection and Error and normally you just want to catch Exceptions.

Reply:
Thanks a lot for the review @justinmclean. Done the required changes.